technic/tools/tree_tap.lua

```diff
@@ -38,8 +46,8 @@ minetest.register_tool("technic:treetap", {
 minetest.register_craft({
 	output = "technic:treetap",
 	recipe = {
-		{"pipeworks:tube_1", "group:wood",    "default:stick"},
-		{"",               "default:stick", "default:stick"}
+		{"pipeworks:tube_1", "group:wood",    "group:stick"},
+		{"",               "group:stick", "group:stick"}
```

Comment:
Better also set these to `materials.stick` to keep it simple and minimize changes to previous behavior (in an attempt to keep focus on MCL compatibility, if recipes for MTG needs to be changed do it in another PR)

Reply:
changed